te_markdown/domainexperts.md

```diff
@@ -1,18 +1,33 @@
 ---
 layout: page
-title: Domain Experts
+title: Test Your Domain
 permalink: /te_markdown/domainexperts/
 ---
 
```

Comment:
Please add

- Estimated reading time: 8 minutes
- Audience: [Curious party], Noob, Somewhat experienced, Experienced, Expert

Reply:
OK, I'm adding the same to `testfirst.md`. But:
* 8 minutes seems awfully long. Does it really take that long to read the text(s)? If so, we might want to shorten it. It shouldn't take more than two or three minutes to read through, in my opinion.
* Those roles are not the final ones (I hope – most readers won't want to be called "noob" :wink ), we need to go back and change them to something more meaningful to potential readers.

Reply:
```suggestion
    Estimated reading time: 3 minutes
    Audience: [Curious party], Newbies, Somewhat experienced, Experienced, Expert
```